<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ipClass</class>
 <widget class="QMainWindow" name="ipClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IP地址查询</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::NetworkWireless"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="placeholderText">
            <string>默认查询本机IP, 有代理则走代理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>130</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::StrongFocus</enum>
        </property>
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="logEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ip.qrc"/>
 </resources>
 <connections/>
</ui>
